certified-connectors/DocuSignDemo/script.csx

```diff
@@ -208,6 +233,34 @@ public class Script : ScriptBase
       }
     }
 
+    if (operationId.Equals("StaticResponseForVerificationTypeSchema", StringComparison.OrdinalIgnoreCase))
+    {
+      var query = HttpUtility.ParseQueryString(context.Request.RequestUri.Query);
+      var verificationType = query.Get("verificationType");
+
+      response["name"] = "dynamicSchema";
+      response["title"] = "dynamicSchema";
+      response["schema"] = new JObject
+      {
+        ["type"] = "object",
+        ["properties"] = new JObject()
+      };
+
+      if (verificationType.Equals("phoneCall", StringComparison.OrdinalIgnoreCase))
```

Comment:
will you be adding code for other verification types like knowlege-based, ID, SMS, access code ?